_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "coSMicQC: A step toward quality control and improvement of morphology profiles"
+author: Jenna Tomkinson
+member: jenna-tomkinson
+tags:
+  - opinions
+  - process
+  - software
```

Comment:
Consider another specific tag here maybe - "quality control"?

Reply:
That sounds good to me, I will add it!